### PR TITLE
[youporn] Add duration

### PR DESCRIPTION
**Pull request by piplongrun**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] [Searched](https://github.com/ytdl-org/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Read [adding new extractor tutorial](https://github.com/ytdl-org/youtube-dl#adding-support-for-a-new-site)
- [x] Read [youtube-dl coding conventions](https://github.com/ytdl-org/youtube-dl#youtube-dl-coding-conventions) and adjusted the code to meet them
- [x] Covered the code with tests (note that PRs without tests will be REJECTED)
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [x] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Add video duration to output

### CODE REVIEW

File: youtube_dl/extractor/youporn.py [hunk 1]
```diff
@@ -153,6 +153,8 @@ def _real_extract(self, url):
         thumbnail = self._search_regex(
             r'(?:imageurl\s*=|poster\s*:)\s*(["\'])(?P<thumbnail>.+?)\1',
             webpage, 'thumbnail', fatal=False, group='thumbnail')
+        duration = int_or_none(self._html_search_meta(
+            'video:duration', webpage, 'duration', default=False, fatal=False))
```
Comment (adrianheine) [inline youtube_dl/extractor/youporn.py:157]: If you have `default`, `fatal` is not used.

**Review by adrianheine — CHANGES_REQUESTED** [review]

File: youtube_dl/extractor/youporn.py (hunk 1 above)
Comment (piplongrun) [reply youtube_dl/extractor/youporn.py]: I did not know, thanks for the tip! I learn something new every time I do something with youtube-dl :) I've removed the `default`, following the line of the rest of the code in this file.

Comment (adrianheine) [reply youtube_dl/extractor/youporn.py]: I know what you mean :) I think `default=None` is actually better (without `fatal=False`). `fatal=False` makes it emit a warning and return `None`, while `default=None` will just silently return `None`. For optional metadata like `duration`, I think it's better not to have a warning.

**Comment by dstftw** [comment]
Add a test.

**Pull request merged**